docs/builder/android.md

```diff
@@ -104,9 +104,16 @@ This step is required for running the production version of your app. If you ski
 
 1. Once you’ve uploaded your .aab file, Google Play re-signs your app. Because of this, you’ll need to update your asset links file.
    
-2. Go to Google Play Console, select your app release then choose `Setup -> App integrity`, then copy your SHA-256 fingerprint.
-   
-3. Then, paste the fingerprint into your assetlinks.json file:
+2. Go to Google Play Console, select your app release then choose `Setup -> App integrity`.
```

Comment:
It's weird, the original issue says they got confused with this line "select your app release" . They said they were looking for "setup -> app integrity" on the home page. so maybe just add a little emphasizing you HAVE to click your app release first or something idk?